CMakeLists.txt

```diff
@@ -1,13 +1,17 @@
 cmake_minimum_required(VERSION 3.5)
 project(cppJNI)
+set(CMAKE_CXX_STANDARD 14)
 
+# Extend CMAKE_PREFIX_PATH with HanaConfig.cmake and Gtest root
```

Comment:
I can't find HanaConfig.cmake in hana repository - what file exactly do you mean?